src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -72,6 +72,63 @@ static class C { }
             Assert.Equal(expectedResults.Data, results);
         }
 
+        [Fact]
+        public async Task TestGetSemanticTokensRange_FullDoc_RazorAsync_StaticClassName()
```

Comment:
This test doesn't actually fail without the fix. I haven't been able to replicate the issue as a unit test, only in real code. I'll probably delete this test after the CI passes just in case.